src/overlays/actors/ovl_En_Ds2n/z_en_ds2n.h

```diff
@@ -2,16 +2,25 @@
 #define Z_EN_DS2N_H
 
 #include "global.h"
+#include "objects/object_ds2n/object_ds2n.h"
 
 struct EnDs2n;
 
 typedef void (*EnDs2nActionFunc)(struct EnDs2n*, GlobalContext*);
 
 typedef struct EnDs2n {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x90];
+    ///* 0x0144 */ char unk_144[0x90];
+    /* 0x144 */ char pad144[0x4C]; // unused collider?
+    /* 0x190 */ SkelAnime skelAnime;
     /* 0x01D4 */ EnDs2nActionFunc actionFunc;
-    /* 0x01D8 */ char unk_1D8[0x5C];
+    /* 0x1D8 */ s16 limbRotTableY[DS2N_LIMB_MAX];
+    /* 0x1FE */ s16 limbRotTableZ[DS2N_LIMB_MAX];
```

Comment:
Oh, these are fidget tables, there's a couple of NPCs in OoT that use them. You can use them to fake an idle animation.